src/pages/index.js

```diff
@@ -105,80 +200,45 @@ function handleAddCardSubmit(inputValues) {
     });
 }
 
-// Function to handle profile edit
 function handleEditProfileSubmit(inputValues) {
-  userInfo.setUserInfo({
-    name: inputValues.title,
-    description: inputValues.description,
-  });
-  editProfilePopup.close();
+  console.log("Edit Profile Input Values", inputValues);
+  api
+    .editProfile(inputValues.title, inputValues.description)
+    .then(() => {
+      userInfo.setUserInfo({
+        name: inputValues.title,
+        description: inputValues.description,
+      });
+      editProfilePopup.close();
+    })
+    .catch((err) => {
+      console.error("Error updating profile:", err);
+    })
+    .finally(() => {
+      editProfilePopup.setLoading(false);
+    });
 }
 
-// Function to create a card
-function createCard(data) {
-  const card = new Card(
-    data,
-    "#card-template",
-    handleImageClick,
-    handleDeleteCard
-  );
-  return card.getView();
+function handleAvatarSubmit(inputValues) {
+  profileAvatarPopup.setLoading(true);
+  api
+    .updateAvatar(inputValues.avatar)
+    .then(() => {
+      userInfo.updateAvatar(inputValues.avatar);
+      profileAvatarPopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors